config/chainbridge-runtime-config.cere.dev.js

```diff
@@ -103,8 +103,8 @@ window.__RUNTIME_CONFIG__ = {
         tokens: [
           {
             address: "0xb8A59CEF67d12C5C75836aEfF1d97943F5A9F662",
-            name: cereTokenName,
-            symbol: cereTokenSymbol,
+            name: "CN",
+            symbol: "CS",
```

Comment:
Can we use more meaningful names here?

Reply:
The same as [here](https://github.com/Cerebellum-Network/chainbridge-ui/pull/131#discussion_r1247759358).